Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:buster
+
+RUN apt-get update
+RUN apt-get -y install curl git rsync \
+    # Node version 14
+    && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+    && apt-get install nodejs
+
+
+RUN git clone https://github.com/codetheweb/MTU-Transfer-Course-Gatherer.git /app/
+WORKDIR /app/
+
+# Install Python dependencies
+
+RUN pip3 install --upgrade pip -r requirements.txt
+RUN python3 ./manage.py makemigrations
+RUN python3 ./manage.py migrate
+RUN python3 ./banwebScrape.py
+RUN python3 ./manage.py custom_db_sync /tmp/writer.csv
+
+ARG DJANGO_SETTINGS_MODULE
+ENV DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
+
+ARG REACT_APP_ENDPOINT
+ENV REACT_APP_ENDPOINT=$REACT_APP_ENDPOINT
+
+ARG PROD_KEY
+ENV PROD_KEY=$PROD_KEY
+
+WORKDIR /app/webapp/
+
+RUN npm install --only=prod
+RUN npm update
+# Fix vulns
+RUN npm audit fix
+RUN npm run build
```

Comment:
It'll probably be better to form this into a multi-stage container so that way the final image is as light as possible with minimal things installed.  It'll also prevent us from committing the heresy that is curl piping into a shell.

The following is an example of how I've done it with Go before, we can also just use a Node14 container due to the building.
https://gist.github.com/Kiro47/f176ca085c9a40fff3be54ad4a07ed77